NEWS.md

```diff
@@ -1,5 +1,19 @@
 # OSBuild - Build-Pipelines for Operating System Artifacts
 
+## CHANGES WITH 27.1:
+
+ * Add history entries to the layers of OCI archives produced by the
+   `org.osbuild.oci-archive` stage. This fixes push failures to quay.io.
+
+ * Include only a specific, limited set of xattrs in OCI archives produced by
+   the `org.osbuild.oci-archive` stage. This is specifically meant to exclude
+   SELinux-related attributes (`security.selinux`) which are sometimes added
+   even when invoking  `tar` with the `--no-selinux` option.
+
+Contributions from: Christian Kellner
```

Comment:
You sure about that? Several commits of other people there since v27.

Reply:
`make release` gives you a template for NEWS.md, btw.

Reply:
(Feel free to let CI finish first, before changing this)

Reply:
This is a branch, only two commits.

Reply:
Well then better not change anything!

Reply:
This branch (RHEL 8.4.0) only contains two commits from Christian.